packages/Chem/src/utils/mol-creation_rdkit.ts

```diff
@@ -12,7 +12,7 @@ export interface IMolContext {
 
 export function isFragment(molString: string) {
   if (isMolBlock(molString)) {
-    return MolfileHandler.getInstance(molString).isFragment();
+    //return MolfileHandler.getInstance(molString).isFragment();
```

Comment:
why is this commented out?

Reply:
@Aleksashka11 , did you see Maria's question?